content/blog/mapr-for-predictive-maintenance.md

```diff
@@ -0,0 +1,318 @@
+---
+title: MapR for Predictive Maintenance
+date: 2022-02-15T05:31:46.014Z
+author: Ian Downard
+authorimage: /img/Avatar1.svg
+tags:
+  - hpe-ezmeral-data-fabric
+---
+**Editor’s Note:** MapR products and solutions sold prior to the acquisition of such assets by Hewlett Packard Enterprise Company in 2019 may have older product names and model numbers that differ from current solutions. For information about current offerings, which are now part of HPE Ezmeral Data Fabric, please visit <https://www.hpe.com/us/en/software/ezmeral-data-fabric.html>
+
+This project is intended to show how to build Predictive Maintenance applications on MapR. Predictive Maintenance applications place high demands on data streaming, time-series data storage, and machine learning. Therefore, this project focuses on data ingest with MapR Streams, time-series data storage with MapR-DB and OpenTSDB, and feature engineering with MapR-DB and Apache Spark.
```

Comment:
Recommend placing the word "work" after "engineering in the last phrase:
"... and feature engineering work with MapR-DB and Apache Spark."

Reply:
"feature engineering" is its own concept just like "time-series data storage"